backend/src/main/kotlin/app/ehrenamtskarte/backend/userdata/database/Schema.kt

```diff
@@ -0,0 +1,33 @@
+package app.ehrenamtskarte.backend.userdata.database
+
+import app.ehrenamtskarte.backend.projects.database.Projects
+import org.jetbrains.exposed.dao.IntEntity
+import org.jetbrains.exposed.dao.IntEntityClass
+import org.jetbrains.exposed.dao.id.EntityID
+import org.jetbrains.exposed.dao.id.IntIdTable
+import org.jetbrains.exposed.sql.javatime.CurrentTimestamp
+import org.jetbrains.exposed.sql.javatime.timestamp
+
+object UserEntitlements : IntIdTable() {
+    val userHash = binary("userHash")
+    val startDate = timestamp("startDate")
+    val endDate = timestamp("endDate")
```

Comment:
Please make these align with the [Schema of the Cards table](https://github.com/digitalfabrik/entitlementcard/blob/main/backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/database/Schema.kt) We use integer numbers (`long`) for the number of days since 1970. There are helper functions to convert dates to and from this format.

Reply:
Could you please tell me why?
For example, it will be less readable if we need to check the dates in the database for some support purposes. 
But what's the idea behind it?

Reply:
> We might as well use the [date type of postgres](https://www.postgresql.org/docs/current/datatype-datetime.html), but I don't know whether exposed supports that.

Yes, it does! I have changed dates to date type